kolibri/plugins/coach_tools/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
It locks any dependencies for this plugin. This plugin has no separate dependencies, hence it is empty.